File: java/project2/src/arsistema/Promedio.java
package arsistema;

public class Promedio {
    public static void main(String[] args) {
        // Declarar variables
        double altura, promedio, sumatoria = 0;
        final int numeroPersonas = 6;
        double maximo = 0;
        double minimo = 0;
        Util.Encabezado("Promedio de las alturas en cm: ", 50);

        for (int i = 0; i < numeroPersonas; i++) {
            altura = Util.leerNumeroDecimal("Ingresa la altura " + (i+1) );
            sumatoria += altura;
            if (i == 0) { // Primera altura
                maximo = altura;
                minimo = altura;
            } 
            if (altura > maximo) {//ordena el maximo
                maximo = altura;
            }    
            if (altura < minimo) {//ordena el minimo
                minimo = altura;
            }




        }
        promedio = sumatoria / numeroPersonas;
        
        Util.linea(50);
        System.out.printf("El promedio de las alturas es %.2f en cm %n", promedio);
        Util.linea(50);
        System.out.printf("La maxima altura es %.2f en metros %n", maximo);
        Util.linea(50);
        System.out.printf("La minima altura es %.2f en metros %n", minimo);
        Util.linea(50);
    }
}

File: python/promedio.py
'''calcular el promedio de estatura de 6 personas y mostrar la estatura maxima y minima '''
from utilidades import *

def main():
    altura: float = 0
    altMax: float = 0
    altMin: float = 0
    nroPersona: int = 6
    promedio: float = 0
    sumatoria: float = 0   
    
    encabezado('ingreso de alturas', 50)
    
    for i in range(nroPersona):
        altura = leerNumero(f"Ingresa la altura {i + 1} en cm", float)
        sumatoria += altura
        
        if i == 0:
            altMax = altura
            altMin = altura
            
        if altura > altMax:
            altMax = altura
            
        if altura < altMin:
            altMin = altura
        
    promedio = sumatoria / nroPersona
    
    linea(50)
    print(f"Promedio de alturas: {promedio:.2f} cm")
    linea(50)
    print(f"Altura maxima: {altMax:.2f} cm")
    linea(50)
    print(f"Altura minima: {altMin:.2f} cm")

main()
